examples/PySDM_examples/seeding/simulation_1d.py

```diff
@@ -0,0 +1,346 @@
+            dt=settings.dt,
+            mpdata_settings=settings.mpdata_settings,
+            advector_of_t=lambda t: settings.rho_times_w(t) * settings.dt / settings.dz,
+            advectee_of_zZ_at_t0=lambda zZ: settings.water_vapour_mixing_ratio(
+                zZ_to_z_above_reservoir(zZ)
+            ),
+            g_factor_of_zZ=lambda zZ: settings.rhod(zZ_to_z_above_reservoir(zZ)),
+        )
+
+        _extra_nz = settings.particle_reservoir_depth // settings.dz
+        _z_vec = settings.dz * np.linspace(
+            -_extra_nz, settings.nz - _extra_nz, settings.nz + 1
+        )
+        self.g_factor_vec = settings.rhod(_z_vec)
+
+        self.builder = Builder(
+            n_sd=settings.n_sd + settings.n_sd_seeding,
+            backend=backend(formulae=settings.formulae),
+            environment=env,
+        )
+        self.builder.add_dynamic(AmbientThermodynamics())
+
+        if settings.enable_condensation:
+            self.builder.add_dynamic(
+                Condensation(
+                    adaptive=settings.condensation_adaptive,
+                    rtol_thd=settings.condensation_rtol_thd,
+                    rtol_x=settings.condensation_rtol_x,
+                    update_thd=settings.condensation_update_thd,
+                )
+            )
+        self.builder.add_dynamic(EulerianAdvection(mpdata))
+
+        self.products = []
+        if settings.precip:
+            self.add_collision_dynamic(self.builder, settings, self.products)
+
+        displacement = Displacement(
+            enable_sedimentation=settings.precip,
+            precipitation_counting_level_index=int(
+                settings.particle_reservoir_depth / settings.dz
+            ),
+        )
+        self.builder.add_dynamic(displacement)
+
+        self.attributes = self.builder.particulator.environment.init_attributes(
+            spatial_discretisation=spatial_sampling.Pseudorandom(),
+            spectral_discretisation=spectral_sampling.ConstantMultiplicity(
+                spectrum=settings.wet_radius_spectrum_per_mass_of_dry_air
+            ),
+            kappa=settings.kappa,
+            collisions_only=not settings.enable_condensation,
+            n_sd=settings.n_sd,  # only initialize with background SDs
+            z_part=settings.z_part,
+        )
+        r_dry, n_in_dv = spectral_sampling.ConstantMultiplicity(
+            spectra.Lognormal(
+                norm_factor=(
+                    settings.seed_particles_per_volume_STP
+                    / settings.formulae.constants.rho_STP
+                ),
+                m_mode=1 * si.um,
+                s_geom=1.4,
+            )
+        ).sample(
+            n_sd=settings.n_sd_seeding
+        )  # TODO #1387: does not have to be the same?
+        v_dry = settings.formulae.trivia.volume(radius=r_dry)
+        self.seeded_particle_extensive_attributes = {
+            "water mass": np.array([0.0001 * si.ng] * settings.n_sd_seeding),
```

Comment:
should we use `equilibrate_wet_radii` rather than prescribing a `water_mass` for the seeds?